src/emqx.app.src

```diff
@@ -1,6 +1,6 @@
 {application, emqx,
  [{id, "emqx"},
-  {description, "EMQX"},
+  {description, "EMQ X"},
```

Comment:
```suggestion
  {description, "EMQX"},
```

Reply:
it's a deliberate change.
it's a revert.